### DISCUSSION TITLE
https

### DISCUSSION THREAD

**Discussion by flash716** [cloudflare/workerd#3050] [Node.js APIs]
Looks like most of the https APIs are mocked, I'm specifically looking for `get` as it's used in a library I would like to use. 🙏

**Comment by jasnell** [comment]
We are unlikely to implement the `node:https` module directly in the runtime and will likely continue to rely on the unenv polyfills to bring these in. An implementation of `node:https` could likely fit on top of the built-in `fetch` implementation but it is also unlikely that we will ever fully support all of the TLS configuration options, lifecycle events, etc as our internal infrastructure uses an entirely different model than Node.js.

**Comment by shchahrykovich** [comment]
> [unenv] https.request is not implemented yet!

I'm using auth0 lib to do auth and see this error. Is it possible to add request support?

**Reply by harveyjing** [reply]
Same as me.

**Reply by Camusama** [reply]
Same

**Comment by TigerC10** [comment]
@jasnell If you're not likely to implement it, can you offer a workaround or polyfill recommendation?  I'm dependent on a library making these calls so I can't make a change in code to do anything else (like fetch).

**Comment by bhysyq** [comment]
I got this error "Error while making request: [unenv] https.request is not implemented yet!. Error code: undefined" while I am making external HTTP request.
